compiler/parser/Makefile

```diff
@@ -38,7 +38,7 @@ clean :
 	$(JSON_TO_SHELL).native $(JSON_TO_SHELL).byte test.err
 
 libdash:
-	cd libdash && ./autogen.sh || ./autogen.sh && ./configure && make && sudo make install
+	cd libdash && ./autogen.sh && ./configure && make
```

Comment:
The reason why I had this is that sometimes it failed spuriously on the first run (I have to check again on my machine whether this happens). We can merge for now and remember this if an issue appears.

Reply:
Noted!